### PR TITLE
PMM-6518 Dashboards datasource name change

### PR DESCRIPTION
**Pull request by adivinho**
[PMM-6518](https://jira.percona.com/browse/PMM-6518)

- [ ] https://github.com/Percona-Lab/pmm-submodules/pull/1162

### CODE REVIEW

**Comment by AlekSi** [comment]
image

Why some lines change data source to `null` and some to `"Metrics"`?

**Comment by adivinho** [comment]
> Why some lines change data source to `null` and some to `"Metrics"`?

Data source 'null' means default. It's used in elements. 
But variables have no default data source.

Screenshot_20201012_131603

File: dashboards/Advanced_Data_Exploration.json [hunk 1]
```diff
@@ -670,7 +670,7 @@
             {
                 "allValue": null,
                 "current": {},
-                "datasource": "Prometheus",
+                "datasource": "Metrics",
```
Comment (atymchuk) [inline dashboards/Advanced_Data_Exploration.json:673]: Can we document cases when the `datasource` takes the value of `null` vs `Metrics`.

**Review by atymchuk — CHANGES_REQUESTED** [review]

**Comment by atymchuk** [comment]
> > Why some lines change data source to `null` and some to `"Metrics"`?
> 
> Data source 'null' means default. It's used in elements.
> But variables have no default data source.

Where do we define that Metrics is the default datasource?

**Comment by adivinho** [comment]
> > > Why some lines change data source to `null` and some to `"Metrics"`?
> > 
> > 
> > Data source 'null' means default. It's used in elements.
> > But variables have no default data source.
> 
> Where do we define that Metrics is the default datasource from now on?

It's done here 
https://github.com/percona/pmm-server/pull/234/files#diff-6d9daaf92d20be62aca45159d3c7d75b85323ed98a6c9cf22fb87c7610acee3fR175

**Review by atymchuk — APPROVED** [review]

**Pull request merged**